### ISSUE TITLE
website backup

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#11914]
We currently depend on eclipse.org/sumo and sumo.dlr.de. The best thing would be to let one work as the backup for the other (which might be impossible given the hugo setup). The old sumo.dlr.de provided by the central DLR IT seems not to work anymore. Another option is to keep a small backup at sumo.sourceforge.net (only the docs and the last release)

**Comment by namdre** [comment]
you mean the ssl-certificate outage of `sumo.dlr.de` is not temporary?

**Comment by behrisch** [comment]
It was temporary, we just do not have a proper backup solution currently. Here is the idea:

- We should generate a copy of the user docs (and maybe pydoc) and upload it to Eclipse.
- This should be updated regularly (every release?) but not daily to avoid unnecessary commits in the website repo.
- Vice versa we should generate a copy of the Eclipse site (run hugo) regularly on our DLR server. This way both servers hold the same content (except for daily reports and releases and maybe not up to date)
- we need another hugo config for the "local" mode which keeps all links on the same server and redirects downloads to sourceforge.

The "strategy" is if Eclipse goes down we simply remove the redirect on sumo.dlr.de and change the redirects on sourceforge and sumo-sim.org to point to sumo.dlr.de instead of Eclipse. If sumo.dlr.de goes down, we switch the hugo configuration on Eclipse to "local" mode. @angelobanse Do you think that is feasible?

**Comment by angelobanse** [comment]
Sounds reasonable and it should be possible. Search engines should not find/list our backup pages (the documentation copy on Eclipse and the landing page backup on sumo.dlr.de) - to avoid multiple/redundant results.

If sumo.dlr.de goes down, would the redirect to Eclipse even work?

We could locate this alternative hugo landing page under sumo.dlr.de/home.

**Comment by behrisch** [comment]
> If sumo.dlr.de goes down, would the redirect to Eclipse even work?

No, probably not. We would need to notify people that they should use eclipse.org/sumo and/or point to the other locations where we can change the redirects (sumo.sf.net and sumo-sim.org).

> We could locate this alternative hugo landing page under sumo.dlr.de/home.

why not directly in the root?

**Comment by angelobanse** [comment]
> why not directly in the root?

I guess we could have the backup landing page in the root (sumo.dlr.de) and keep the current (permanent) redirect to Eclipse (sumo.dlr.de -> eclipse.org/sumo), making this backup not reachable until we disable the redirect.

**Comment by behrisch** [comment]
Add Links to alternative documentation sources to the README

**Comment by behrisch** [comment]
sumo.dlr.de now does a daily build of the sumo website into sumo.dlr.de/hugo. In case of a failure of the eclipse site it should suffice to use htaccess.dlr instead of htaccess. We should still do a cleanup of the sumo.dlr.de server some time
